open_academy/models/open_academy_session.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields
+
+class OpenAcademySession(models.Model):
+    _name = "open.academy.session"
+
+    name = fields.Char(required=True)
+    start_date = fields.Date()
+    duration = fields.Float(digits=(6,2), help="Duration in days")
+    seats = fields.Integer(string="Number of seats")
+
+    instructor_id = fields.Many2one("res.partner")
+    course_id = fields.Many2one("open.academy.course", ondolete="cascade", required=True)
+    attende_id = fields.Many2many("res.partner", string="Attendes")
```

Comment:
```suggestion
    attendee_ids = fields.Many2many("res.partner", string="Attendes")
```

See [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields) for fields